### DISCUSSION TITLE
posts/when-pulumi-met-nuke

### DISCUSSION THREAD

**Discussion by giscus** [TechWatching/techwatching.dev#39] [Announcements]
# posts/when-pulumi-met-nuke

https://www.techwatching.dev/posts/when-pulumi-met-nuke

**Comment by vmandic** [comment]
Brilliant article, such a powerful combo to be aware of. Thanks for writing this down.

**Reply by TechWatching** [reply]
Thanks! Definitely a nice combo.

**Comment by kaylumah** [comment]
Very nice overview, looking forward to testing out Nuke myself

**Reply by TechWatching** [reply]
I am not a Nuke expert but I found the developer experience to be great. I'm sur you will love it.

**Comment by janban-outlook** [comment]
I prefer to make the Deploy target dependent on ProvisionInfra. I am trying to understand why you didn't.

**Reply by TechWatching** [reply]
I could have, and to be honest I don't remember why I didn't when I wrote this article. I guess I chose to use `After` instead of `DependsOn` to be able to deploy my application without the ProvisionInfra target.
